plugin/pkg/auth/authorizer/rbac/bootstrappolicy/namespace_policy.go

```diff
@@ -93,6 +93,15 @@ func init() {
 			eventsRule(),
 		},
 	})
+	addNamespaceRole(metav1.NamespaceSystem, rbac.Role{
+		// role for the leader locking on supplied configmap
+		ObjectMeta: metav1.ObjectMeta{Name: "system::leader-locking"},
+		Rules: []rbac.PolicyRule{
```

Comment:
> @deads2k looking for .Names("kube-controller-manager", "kube-scheduler") ?
> 
> or are you looking for separate rules for each one? I'm new to this area of the code so this is also a learning experience for me atm.

I think that separate roles are the best way at the moment.  If we start developing more of these, I could see trying to create function which generates the `system:leader-locking-<name>` and binding pairs.  The additional work to create two roles and bindings doesn't seem overly onerous. 

To expand on my concern, there is a configmap which contains a CA bundle used by aggregated API servers to verify front proxies to allow zero-config in aggregated API servers.  The CA bundle itself is not confidential information, but If a bad actor were able to modify that CA bundle, they could trick aggregated API servers into trusting "evil-proxy".  By limiting the update power of your leasers, you close off an avenue for attack.